spec/models/user_spec.rb

```diff
@@ -12,16 +12,16 @@
   end
 
   it "#role is :user by default" do
-    expect(@user.role).to eq :user
+    expect(@user.role).to eq "user"
```

Comment:
I'm really splitting hairs here, but I think the description should read, "#role returns 'user' by default". My reasoning is that I'd be super confused if I saw a spec failing that expected a symbol when really it needed to provide a string.

Like I mentioned on the other PR, this is a thing with Devise roles that makes me squirm a bit, so I understand how it's basically all the same here. (And, of course, the roles themselves are an enum and so stored as ints in the database, which is fine but just, like, Devise, what were you actually thinking?)

Anyway. Let's rewrite the whole thing in something type-safe like Haskell. (j/k j/k)